test/lib.fetching.bots.subscribe-bot.test.js

```diff
@@ -0,0 +1,56 @@
+require('./init');
+var expect = require('chai').expect;
+var SubscribeBot = require('../lib/fetching/bots/subscribe-bot');
+var Bot = require('../lib/fetching/bot');
+var contentServiceFactory = require('../lib/fetching/content-service-factory');
+var Source = require('../models/source');
+var request = require('supertest');
+
+describe('Subscribe bot', function() {
+  before(function(done) {
+    var source = new Source({nickname: 't', media: 'smsgh', keywords: 'bozo'});
+    var contentService = contentServiceFactory.create(source);
+    subscribeBot = new SubscribeBot({source: source, contentService: contentService});
+    // console.log(subscribeBot);
+    done();
+  });
+
+  it('should instantiate a subscribe-type content service', function() {
+    expect(subscribeBot.type).to.equal('subscribe');
+    expect(subscribeBot).to.be.instanceOf(Bot);
+  });
+
+/*
+  it('should tell content service to start streaming reports', function(done) {
```

Comment:
No. This test was discarded when we decided not to write a test which would check if SMSGh selectively stops listening to one source while listening to others.